development/setup/amplifysetup.md

```diff
@@ -1 +1,104 @@
 # Owner - Kamal
+
+# Setting Up an Amplify Full-Stack App
+This guide walks you through creating a full-stack application using Amplify. We'll cover the essential steps to get your project up and running.
+
+## Prerequisites:
+
+An AWS account with appropriate permissions.
+Node.js and npm (or yarn) installed on your development machine.
+Basic understanding of web development concepts (HTML, CSS, JavaScript).
+
+## Steps:
+ 
+### 1.Install the Amplify CLI:
+
+### Open a terminal window and run the following command:
+npm install -g @aws-amplify/cli
+
+### 2.Initialize a New Project:
+
+Navigate to your desired project directory and run:
+amplify init
+
+Follow the prompts to choose a project type (e.g., web app) and configure authentication settings (if applicable).
+
+### 3.Define Your App Structure:
+
+Amplify uses a directory structure to manage your project. It creates essential folders for your backend (API), frontend (client), and cloud resources.
+
+### 4.Configure Your Backend (API):
+
+Use the Amplify CLI to define your backend schema:
+
+amplify add schema
+
+## setting up amplify
+### run these commands
+
+amplify configure
+press ENTER (x2)
+
+Enter the access key of the newly created user: 
+
+accessKeyId: (put_your_access_ID)
+ secretAccessKey:  (put_your_access_KEY)
+ (NOTE: these details will be provided by you admin)
+
+ Profile Name:  default
+(terminal will get exit() , run below commands)
+ amplify pull
+
+ Select the authentication method you want to use: AWS profile
+ Please choose the profile you want to use default  
+ Which app are you working on? (choose_your_app_name)
+ Choose your default editor: Visual Studio Code     
+Choose the type of app that you're building · javascript
+
+### Please tell us about your project
```

Comment:
can you add a note at start as well telling that there are changes in Distribution Directory path, do take note of that... Generally people have a habbit of skimming through...